utils/jwt.py

```
@@ -0,0 +1,46 @@
+from config.config import settings
+
+# JWT 토큰 검증 함수 (만료/유효성)
+def verify_token(token: str):
+    try:
+        key = base64.b64decode(settings.jwt_secret)
+        payload = jwt.decode(token, key, algorithms=["HS256"])
+        return payload
+    except jwt.ExpiredSignatureError:
+        HTTPException(status_code=401, detail="Expired token") # 만료된 토큰
+    except jwt.InvalidTokenError:
+        raise HTTPException(status_code=401, detail="Invalid token") # 잘못된 토큰
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)